src/backend/core/management/commands/index.py

```diff
@@ -0,0 +1,41 @@
+"""
+Handle search setup that needs to be done at bootstrap time.
+"""
+
+import logging
+import time
+
+from django.core.management.base import BaseCommand, CommandError
+
+from core.services.search_indexers import get_file_indexer
+
+logger = logging.getLogger("drive.search.bootstrap_search")
+
+
+class Command(BaseCommand):
+    """Index all files to remote search service"""
+
+    help = __doc__
+
+    def handle(self, *args, **options):
+        """Launch and log search index generation."""
+        indexer = get_file_indexer()
+
+        if not indexer:
+            raise CommandError("The indexer is not enabled or properly configured.")
```

Comment:
If there is no indexer, only an info message should be printed informing that there is no indexer service configured ? I don't think it should be an error